api/app/src/external/commonwell/document/document-query.ts

```diff
@@ -86,6 +85,14 @@ export async function queryAndProcessDocuments({
       // send webhook to cx async when docs are done processing
       processPatientDocumentRequest(organization.cxId, patient.id, toDTO(FHIRDocRefs));
 
+      if (processingToFhir) {
```

Comment:
Wouldn't this be the opposite, only mark as completed after downloaded if there's nothing being converted to FHIR?

Reply:
Removed with update